exp/wsj/write_hdf_dataset.sh

```diff
@@ -94,11 +94,11 @@ $LVSR/bin/kaldi2fuel.py $h5f add_text --applymap tmp_chars.txt tmp_chars_all cha
 compute-fbank-feats --use-energy=true --num-mel-bins=40 "scp:tmp_convert_all_wav.scp" 	"ark:-" | \
 	add-deltas ark:- ark,scp:tmp_fbank40.ark,tmp_fbank40.scp
 
-compute-global-cmvn-stats.py \
+scripts/compute-global-cmvn-stats.py \
```

Comment:
Actually, it should be without `scripts`. We install this script [here](https://github.com/dmitriy-serdyuk/kaldi-python/blob/master/setup.py#L20).